bin/apisix

```diff
@@ -560,6 +560,26 @@ local function init_etcd(show_output)
     for index, host in ipairs(yaml_conf.etcd.host) do
         local is_success = true
 
+        local token_head = ""
+        if etcd_conf.user and etcd_conf.password then
+            local uri_auth = host .. "/v3/auth/authenticate"
+            local json_auth = {
+                name =  etcd_conf.user,
+                password = etcd_conf.password
+            }
+            local post_json_auth = dkjson.encode(json_auth)
+            local cmd_auth = "curl -s " .. uri_auth .. " -X POST -d '" .. post_json_auth
+                             .. "' --connect-timeout " .. timeout
+                             .. " --max-time " .. timeout * 2 .. " --retry 1 2>&1"
+
+            local res_auth = execute_cmd(cmd_auth)
```

Comment:
Should be `util.execute_cmd`

Reply:
anything else? 1 failing checks

Reply:
@jxhecong
The check failed because `execute_cmd` doesn't exist (it should be `util.execute_cmd`). Let's fix it and see what's next.